src/Chainweb/Version/Mainnet.hs

```diff
@@ -141,7 +141,7 @@ mainnet = ChainwebVersion
         Chainweb219Pact -> AllChains (ForkAtBlockHeight $ BlockHeight 3_774_423) -- 2023-06-02 00:00:00+00:00
         Chainweb220Pact -> AllChains (ForkAtBlockHeight $ BlockHeight 4_056_499) -- 2023-09-08 00:00:00+00:00
         Chainweb221Pact -> AllChains (ForkAtBlockHeight $ BlockHeight 4_177_889) -- 2023-10-20 00:00:00+00:00
-        Chainweb222Pact -> AllChains ForkNever -- TODO: fill in
+        Chainweb222Pact -> AllChains (ForkAtBlockHeight $ BlockHeight 4_335_753) -- 2023-12-14 00:00:00+00:00
```

Comment:
Should there be a `Chainweb223Pact` that returns `ForkNever`?

Reply:
I actually really like the idea of handling that this release. Sure, add it in.

Reply:
4335753 - 4280488 (current mainnet height) = 55265.
Close to the testnet number below.

Reply:
Works for me